tools/HttpExceptions/GenerateClasses.php

```diff
@@ -36,18 +34,8 @@ class <?= $className ?> extends HttpException {
     public const DESCRIPTION = "<?= $code->description ?>";
     public const HREF = "<?= $code->mdn ?>";
 }
-
 <?php 
 
-file_put_contents("../src/Exceptions/$className.php", ob_get_clean());
-
-
-
-echo TAB.TAB."Exceptions\\$className::class => []";
-echo ($key !== count($codes)-1) ? ",\n" : "\n";
-
-}
-
-echo TAB."];";
+file_put_contents("../../src/Exceptions/$className.php", ob_get_clean());
 
-echo "\nBE SURE TO UPDATE 'src/Router.php:32'";
\ No newline at end of file
+}
```

Comment:
File should have a unix line ending it can cause very weird issues.

Just use https://marketplace.visualstudio.com/items?itemName=jkiviluoto.tws you can use this to add them automatically. It also removes trailing whitespaces